static/include/snippet/visualize.md

```diff
@@ -0,0 +1,16 @@
+You can visualize your machine's geometries and frames in the Viam app:
+
+{{% alert title="Tip" color="tip" %}}
+To visualize a component without moving any real-world hardware, configure a `fake` model to represent your component (for example, a [`fake` arm](/operate/reference/components/arm/fake/)), then follow the steps below using the fake component.
+{{% /alert %}}
```

Comment:
I think move this to the end of the page. I'd expect most people would want to use this with real things?